airflow/operators/sensors.py

```diff
@@ -52,10 +52,10 @@ def poke(self, context):
         raise AirflowException('Override me.')
 
     def execute(self, context):
-        started_at = datetime.now()
+        self.started_at = datetime.now()
```

Comment:
AFAICT your current test case doesn't need this (just needs a self.started_at = None in TimeoutTestSensor's constructor)